inc/db.function.php

```diff
@@ -636,7 +636,7 @@ function getEntitiesRestrictRequest(
  * @param string $field             field where apply the limit (id != entities_id) (default '')
  * @param mixed $value              entity to restrict (if not set use $_SESSION['glpiactiveentities']).
  *                                  single item or array (default '')
- * @param boolean $is_recursive     need to use recursive process to find item
+ * @param boolean|string $is_recursive     need to use recursive process to find item
```

Comment:
```suggestion
 * @param boolean|'auto' $is_recursive     need to use recursive process to find item
```
This would be the most accurate